Klassenserver/src/me/marcel/klassenserver/config/ConfigEditor.java

```diff
@@ -74,13 +76,30 @@ public boolean getBoolean(String path) {
 	public List<String> getStringList(String path) {
 		return this.getConfig().getStringList(path);
 	}
-	
+	public Location getLocation(String path) {
+		String world = ConfigManager.editor("routes").getString(path+".world");
+		double x = ConfigManager.editor("routes").getDouble(path+".x");
+		double y = ConfigManager.editor("routes").getDouble(path+".y");
+		double z = ConfigManager.editor("routes").getDouble(path+".z");
+		float pitch = (float) ConfigManager.editor("routes").getDouble(path+".pitch");
+		float yaw = (float) ConfigManager.editor("routes").getDouble(path+".yaw");
+		return new Location(Bukkit.getWorld(world), x, y, z, yaw, pitch);
+	}
 	public File getFile() {
 		return this.file;
 	}
 	
 	public FileConfiguration getConfig() {
 		return this.config;
 	}
+
+	@SuppressWarnings("null")
+	public List<Location> getCheckpoints(String path) {
+		List<Location> checkpoints = null;
+		for(String checkpoint : this.getConfig().getStringList(path)){
+			checkpoints.add(ConfigManager.editor("routes").getLocation(path+"."+checkpoint));

```

Comment:
Stimmt das so? Path+.+checkpoint?

Reply:
Ja, aber du musst die List<Location> initialiseren, sonst kannst du ja nichts hinzufügen oder so..

List<Location> checkpoints = new ArrayList<Location>();

Reply:
Achso ok aber ansonsten stimmt alles so?

Reply:
Deswegen muss man die Punkte machen:

Beispiel (config.yml):

settings:
  servername: 'Klassenserver'

In dem Fall wäre dann der Pfad => "settings.servername"

Reply:
ich war mir nicht sicher ob bei getStringList(path) path mit wieder zurück übergeben wird also das die liste dann ist
-path."was da gefunden wurde"
-path."das nächste was da gefunden wurde"


hab die initialisierung hinzugefügt